src/Agent.Worker/Build/BuildCommandExtension.cs

```diff
@@ -16,6 +16,8 @@ public sealed class BuildCommandExtension : AgentService, IWorkerCommandExtensio
 
         public string CommandArea => "build";
 
+        public string SupportedHostType => string.Empty;
```

Comment:
@TingluoHuang - I prefer keeping it as SupportedHostType. That way we don't need to keep the list of unsupported host types in sync when we add a new host type where build.* commands wouldn't be supported for e.g. As such I don't expect unsupported command types in a specific host to grow.